src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -127,9 +127,10 @@ private Uri CreateAuthorizationUri(bool addVerifier = false, bool addState = fal
             if (addVerifier)
             {
                 _codeVerifier = PlatformPlugin.CryptographyHelper.GenerateCodeVerifier();
-                string codeVerifierHash = PlatformPlugin.CryptographyHelper.CreateSha256Hash(_codeVerifier);
+                string codeVerifierHash = MsalHelpers.EncodeToBase64Url(
```

Comment:
I don't understand why the change is made. Actually i prefer the code before the change. Generate the code verifier hash first, and then encoded it when we set the request params, this looks more reasonable to me, and help on debugging.